oluwajuwon/Fast-Food-Fast
# Feature user signup #160807932

### What does this pull request do?
This pull request creates the sign up authentication endpoint for fast food fast

### Description of  the Task to be completed?
Users should be able to create an account with their details to be able to order for food on fast food fast. 
This feature creates an endpoint for creating an account

### How should this be manually tested?
Users can test it by:
  - Install git
  - Open up the command line
  - Run `git clone https://github.com/oluwajuwon/Fast-Food-Fast.git` to clone the repo
  - Run `npm install` to install all package dependencies
  - Run `npm run start` to start the app
  - Downloading the app Postman from - https://www.getpostman.com/ and install it
  - Open up and postman and enter the URLs below:
http://localhost:3000/api/v1/auth/signup - using the POST request (select the request
by the left side of the URL bar) to create a new account and enter all the required fields (username, full name, email, password and confirm password)
  - If it returns the status code 201 and a message that the sign up was successful, then it works fine

### What are the relevant pivotal tracker stories?
The relevant pivotal story is:
 - A user should be able to create an account - https://www.pivotaltracker.com/story/show/160807932
